test/images/image-util.sh

```diff
@@ -262,6 +262,15 @@ if [[ "${WHAT}" == "all-conformance" ]]; then
   for image in "${conformance_images[@]}"; do
     eval "${TASK}" "${image}" "$@"
   done
+elif [[ "${WHAT}" == "updated-images" ]]; then
+  # if updated-images is selected it obtains the images modified respect HEAD and updates them
+  # this is required for the postsubmit job used to update the statging repo and not having to
+  # rebuild all the images
+  shift
+  updated_images=$(git diff --dirstat=files,0 HEAD~1 -- "${KUBE_ROOT}"/test/images/* | cut -d "/" -f 3)
```

Comment:
How would this handle a batch of 5 PR's merging, and HEAD^5 was the PR that modified test/images?

Reply:
yeah, that is my question for the git experts https://github.com/kubernetes/kubernetes/pull/96118#discussion_r516148331, but I think that this should be the right behavior, other thing is that is tehnically possible :shrug:

Reply:
duh, this is always targeting master